fxmanifest.lua

```diff
@@ -4,6 +4,11 @@ game 'gta5'
 description 'qb-bossmenu'
 version '2.0.0'
 
+shared_scripts {
+    '@qb-core/shared/locale.lua',
+    'locales/*.lua',
```

Comment:
You need to load the `en.lua` first since it should be the fallback lang
```lua
shared_scripts {
    '@qb-core/shared/locale.lua',
    'locales/en.lua',
    'locales/*.lua',
}
```